timur/lib/client/jsx/hooks/useRequestDocuments.jsx

```diff
@@ -0,0 +1,38 @@
+import {useCallback} from 'react';
+import {useNotifications} from "etna-js/components/Notifications";
+import {useActionInvoker} from "etna-js/hooks/useActionInvoker";
+import {requestDocuments} from "../actions/magma_actions";
+import {documentOnWaiver, isWaiverPatientModel} from "../utils/patients";
+
+export function useRequestDocuments() {
```

Comment:
"Wraps" the requestDocuments action with additional logic that is aware of the component context in which it is being used.  In the future, I think we should move away from using actions to dispatch side effects, and use React hooks instead.  But we'll get there bit by bit.